packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -166,6 +167,7 @@ exports.sourceNodes = async (
     params = {},
     concurrentFileRequests = 20,
     concurrentAPIRequests = 20,
+    requestTimeoutMS = 30000,
```

Comment:
@KyleAMathews - is there a sensible maximum to enforce here?

Reply:
This is what the timeout has been for some time so it seems to meet most people's needs

Reply:
You shouldn't need to set a default value, pluginOptions should be set correctly through pluginOptionsSchema